js/views/shareScreen.js

```diff
@@ -45,7 +46,10 @@ var ShareScreen = React.createClass({
     return (
       <div className="share-screen">
         <SharePanel {...this.props} />
-        <CloseButton closeAction={this.handleClose} fontStyleClass={this.props.skinConfig.icons.dismiss.fontStyleClass} />
+        <CloseButton closeAction={this.handleClose} fontStyleClass={this.props.skinConfig.icons.dismiss.fontStyleClass}
+          fontFamilyName={this.props.skinConfig.icons.dismiss.fontFamilyName}
```

Comment:
I monkey patched this into the closeButton component, now that we have the Icon component, it might make sense to just refactor the close button into some kind of Icon, but this pull is already large enough that it's probably better off as a different PR.